webapp/TargetedMS/js/QCTrendPlotPanel.js

```diff
@@ -858,7 +867,27 @@ Ext4.define('LABKEY.targetedms.QCTrendPlotPanel', {
                     {
                         this.metric = newVal;
                         this.havePlotOptionsChanged = true;
+                        var items = this.otherPlotOptionsToolbar.items.items;
+                        var showAllSeriesCheckBoxLocation;
+
+                        for(var i=0; i<items.length;i++ ) {
+                            if(items[i].boxLabel ==='Show All Series in a Single Plot')
+                                showAllSeriesCheckBoxLocation = i;
+                        }
 
+                        if(this.getMetricPropsById(newVal).precursorScoped) {
```

Comment:
Using setVisible() would be a little less code than needing a separate sets of hide() and show() calls.